### PR TITLE
feat(compute): add samples for suspend/resume

### PR DESCRIPTION
**Pull request by FrodoTheTrue**

### CODE REVIEW

File: compute/instances/suspend-resume/resume.go [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+		Instance: instanceName,
+	}
+
+	instance, err := instancesClient.Get(ctx, getInstanceReq)
+	if err != nil {
+		return fmt.Errorf("unable to get instance: %v", err)
+	}
+
+	if instance.GetStatus() != "SUSPENDED" {
+		// f"Only suspended instances can be resumed. "
```
Comment (muncus) [inline compute/instances/suspend-resume/resume.go:55]: looks like some leftover comments which can be removed.

File: compute/instances/suspend-resume/resume.go [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+	"fmt"
+	"io"
+
+	compute "cloud.google.com/go/compute/apiv1"
+	computepb "google.golang.org/genproto/googleapis/cloud/compute/v1"
+)
+
+// resumeInstance resumes a suspended Google Compute Engine instance
+// (with unencrypted disks).
+func resumeInstance(
```
Comment (muncus) [inline compute/instances/suspend-resume/resume.go:29]: This indentation style seems inconsistent with other samples. While I dont see any explicit guidance on function signatures, most others are on a single line.

File: compute/instances/suspend-resume/suspend_resume_test.go [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+				{
+					Name: proto.String("default"),
+				},
+			},
+		},
+	}
+
+	op, err := instancesClient.Insert(ctx, req)
+	if err != nil {
+		t.Errorf("unable to create instance: %v", err)
```
Comment (muncus) [inline compute/instances/suspend-resume/suspend_resume_test.go:95]: This should be Fatalf(), since we cannot continue the test without a VM.

**Review by muncus — CHANGES_REQUESTED** [review]

**Comment by FrodoTheTrue** [comment]
@muncus could you please take a look once more? I've fixed the comments

**Review by muncus — APPROVED** [review]
Thanks for the PR, Fedor!

**Pull request merged**